app/gui/src/main/scala/org/bitcoins/gui/dlc/dialog/SignDLCDialog.scala

```diff
@@ -1,190 +1,220 @@
 package org.bitcoins.gui.dlc.dialog
 
 import org.bitcoins.cli.CliCommand._
-import org.bitcoins.core.protocol.dlc.models.{DLCStatus, EnumContractDescriptor}
 import org.bitcoins.core.protocol.tlv._
-import org.bitcoins.gui.GlobalData
-import org.bitcoins.gui.dlc.GlobalDLCData.dlcs
-import org.bitcoins.gui.util.GUIUtil
-import scalafx.Includes._
-import scalafx.geometry.{Insets, Pos}
-import scalafx.scene.control._
-import scalafx.scene.layout._
-import scalafx.stage.Window
 
-import scala.collection._
-import scala.util.{Failure, Success, Try}
+import org.bitcoins.gui.dlc.GlobalDLCData
+
+import scalafx.scene.Node
+
```

Comment:
this removes the data appearing for the sign message, is that what we want?

Reply:
I'm also a bit confused about what is happening here, this is for the sign not the add sigs and broadcast flow?

Reply:
oh I see that it is to add file stuff to sign but it also gets rid of existing deserialization logic instead of being an alternative, @benthecarman could you maybe make a quick fix for this?